modules/social_features/social_event/modules/social_event_managers/social_event_managers.module

```diff
@@ -65,6 +65,40 @@ function social_event_managers_activity_send_email_notifications_alter(array &$i
   }
 }
 
+/**
+ * Implements hook_entity_access().
+ *
+ * This allows view access to event_enrollment entities for users which are the
+ * recipients of the event enrollment but not the owner of the entity. For
+ * example a site manager can create an enrollment for a specific user.
+ */
+function social_event_managers_event_enrollment_access(EventEnrollment $event_enrollment, $operation, AccountInterface $account) {
+  if ($operation != 'view') {
```

Comment:
Please add some comments on why we are returning `AccessResult::neutral();` if we are viewing the content